tests/src/test/scala/common/StreamLogging.scala

```diff
@@ -32,4 +33,6 @@ trait StreamLogging {
     val stream = new ByteArrayOutputStream
     val printstream = new PrintStream(stream)
```

Comment:
Why would you need to override those? They are test convenience already? If you need to override them, you might as well write your own helper?